### DISCUSSION TITLE
I upgraded to pro but storage is still limited

### DISCUSSION THREAD

**Discussion by sbkobaidze** [vercel/community#4451] [Bug Report]
### Page to Investigate

https://vercel.com/dashboard/

### Steps to Reproduce

I already created support ticket but nobody has answered me for days and this is urgent because it affects production. I also tried to create another database and connect to it but still it says that endpoints is unavailable

**Comment by amyegan** [comment]
Hi @sbkobaidze. If you share the ticket number, I can look into it for you 🙏

**Comment by sbkobaidze** [comment]
@amyegan Hey , #00164837 thanks

**Reply by amyegan** [reply]
Thanks for sharing that. It looks like you didn't include much additional info on the ticket, but it would be very helpful to know more. 

Can you share screenshots or a video to help us see what you tried and what's going wrong? You can add the extra info to your ticket or post it here (just be sure to exclude sensitive info if publicly posting here).

**Reply by sbkobaidze** [reply]
Sure, so I had overdue to payment and storage got limited. I paid the payment and storage is still limited but it still asks me to upgrade to pro, but I am Pro subscription tier right now - https://imgur.com/a/OP2WBcV here's the screenshot. The strange thing is that I created another database and tried to connect my project to it, can not make connection there too.

**Reply by amyegan** [reply]
The limits associated with your database match the Pro plan now. The engineering team put some changes in place to fix this bug so it won't cause problems for anyone else. Thank you for reporting it!

**Reply by sbkobaidze** [reply]
@amyegan Thanks!
